src/components/common/ListPeopleAndRoles.vue

```diff
@@ -77,7 +77,7 @@
 
         <!-- List Content -->
         <v-row
-          v-for="(orgPerson, index1) in personList"
+          v-for="(orgPerson, index1) in filteredPersonList"
```

Comment:
filtered person list to exclude Director role see line 271). This is done to avoid using v-if with v-for
Ref : https://mokkapps.de/vue-tips/dont-use-v-if-with-v-for